src/status_im2/contexts/chat/messages/drawers/view.cljs

```diff
@@ -195,3 +198,62 @@
               :on-press            (fn []
                                      (rf/dispatch [:hide-bottom-sheet])
                                      (when on-press (on-press)))}]))]])))
+
+(defn get-reaction-authors-accessibility-id
+  [reaction-type]
+  (-> reaction-type
+      constants/reactions
+      name
+      (str "-reaction-count")))
+
+(defn reaction-authors
+  [reaction-authors show-reaction-author-list?]
+  (let [selected-tab (reagent/atom (first (keys reaction-authors)))]
+    [:f>
+     (fn []
+       (rn/use-effect (fn []
+                        (fn []
+                          (reset! show-reaction-author-list? false)))
+                      [reaction-authors])
+       [:<>
+        [quo/tabs
+         {:style          style/tabs
+          :size           32
+          :on-change      #(reset! selected-tab %)
+          :default-active @selected-tab
+          :data           (mapv (fn [[reaction-type author-details]]
+                                  ^{:key (str reaction-type)}
+                                  {:id                  reaction-type
+                                   :style               style/tabs
+                                   :label               [rn/view {:style style/tab}
+                                                         [quo/icon
+                                                          (get constants/reactions reaction-type)
+                                                          {:no-color        true
+                                                           :container-style style/tab-icon}]
+                                                         [quo/text
+                                                          {:weight :medium
+                                                           :size   :paragraph-1
+                                                           :style  (style/tab-count (= @selected-tab
+                                                                                       reaction-type))}
+                                                          (count author-details)]]
+                                   :accessibility-label (get-reaction-authors-accessibility-id
+                                                         reaction-type)})
+                                reaction-authors)}]
+        [rn/scroll-view
+         {:style {:height 320
+                  :flex   1}}
+         (doall
+          (map (fn [{:keys [from compressed-key]}]
```

Comment:
The anonymous function for `map` here is unnecessary and can be extracted to a separate var.

Reply:
Done